types/wordpress__components/index.d.ts

```diff
@@ -2,6 +2,7 @@
 // Project: https://github.com/WordPress/gutenberg/tree/master/packages/components/README.md
```

Comment:
This change to `isSecondary` seems to be from `9.0.0`: https://unpkg.com/browse/@wordpress/components@9.0.0/src/button/README.md

Will you bump the version in line 1 in this file to reflect that?

Reply:
Thanks for catching that!